test/library/packages/Zarr/EXECENV

```diff
@@ -1,3 +1,4 @@
 #!/bin/bash
 LD_LIBRARY_PATH=$LD_LIBRARY_PATH:.
 DYLD_LIBRARY_PATH=$DYLD_LIBRARY_PATH:.
+MACOS_DEPLOYMENT_TARGET=12.4
```

Comment:
Does this specify a minimum version for MacOS, if so then this seems okay. If not then we should use some sort of script to automatically detect which version we need to target.

Reply:
Looks like it the first case here, so we're good.

Reply:
This was a good catch, I both misnamed the environment variable and put it in the wrong ENV file.